CriticalMass/MapErrorView.xib

```diff
@@ -0,0 +1,76 @@
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="System colors in document resources" minToolsVersion="11.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleToFill" id="iN0-l3-epB" customClass="MapErrorView" customModule="CriticalMaps" customModuleProvider="target">
+            <rect key="frame" x="0.0" y="0.0" width="158" height="30"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <visualEffectView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="J3y-OW-QDP">
+                    <rect key="frame" x="0.0" y="0.0" width="158" height="30"/>
+                    <view key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" id="qOK-Dd-pMy">
+                        <rect key="frame" x="0.0" y="0.0" width="158" height="30"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <stackView opaque="NO" contentMode="scaleToFill" spacing="8" translatesAutoresizingMaskIntoConstraints="NO" id="NuX-ci-7ej">
+                                <rect key="frame" x="8" y="0.0" width="142" height="30"/>
+                                <subviews>
+                                    <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="alert" translatesAutoresizingMaskIntoConstraints="NO" id="e1h-2s-YVe">
+                                        <rect key="frame" x="0.0" y="0.0" width="18" height="30"/>
+                                        <color key="tintColor" systemColor="systemYellowColor"/>
+                                        <constraints>
+                                            <constraint firstAttribute="height" constant="30" id="4rT-59-env"/>
+                                            <constraint firstAttribute="width" constant="18" id="Mvr-o7-SZJ"/>
+                                        </constraints>
+                                    </imageView>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="ServerError" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontForContentSizeCategory="YES" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="xGR-b9-PgM">
```

Comment:
Maybe we should add a `numberOfLines = 0` for this label. We set the error message in this label dynamically and with dynamic type we could reach a display limit with one line.

Reply:
> Beside the error message itself it's recommended to give an action. In this case it's "Please check/come back later". Else we leave the user with a non working map and no idea what to do about it. So the size of the box has to be bigger to fit a proper instruction.
> 
> Also the subtle style may work at a desk. Many people check the app while doing a super quick stop or even while riding. A strong "in your face" style makes it easy to quickly realise that something isn't working. Even without reading. So I recommend the bolder red style as in the design.

Since it stays there until there is no error any more I thought it is nicer to have a subtle warning.
I can also make another version with the bigger message and we could A/B test it

Reply:
currently I have to find a solution for the container that is not growing... But yes. would make sense I guess